docker-compose.yml

```diff
@@ -1,11 +1,12 @@
 services:
-    db:
+    pg:
         image: postgres:14.1-alpine
+        container_name: postgres
         restart: always
         ports:
             - 5432:5432
         env_file:
-            - ./apps/api/.env
+            - ".env.backend"
```

Comment:
undo to the old .env file in the api repo

Reply:
don't understand what you mean

Reply:
keep `./apps/api/.env` as is .. don't replace it

Reply:
both files are used not only one
I don't understand the way you use those files on the project so I want to see how it works in  a meeting since there're multiple .env files and each one has different values